assignments/prototypes.js

```diff
@@ -16,13 +16,39 @@
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
 
+function gameObject(features) {
```

Comment:
it is a useful convention to always use CapitalCase when declaring Constructor functions. this has no effect on the way the code runs, but it provides an important visual cue that makes your code easier to read and understand to other developers and yourself.

```function GameObject(features) {```